prediction_market_agent_tooling/markets/manifold/manifold.py

```
@@ -158,3 +158,17 @@ def get_user_url(cls, keys: APIKeys) -> str:
+    @staticmethod
+    def verify_operational_balance(api_keys: APIKeys) -> bool:
+        """
+        Verify that the agent has at least 1.0 mana balance
+        """
+        MIN_MANA_BALANCE = 1.0
+        user = get_authenticated_user(api_keys.manifold_api_key.get_secret_value())
+        current_balance = user.balance
+        if current_balance >= MIN_MANA_BALANCE:
+            return True
+        else:
+            raise ValueError(f"Insufficient Manifold balance: {current_balance} mana.")           
```

Ruff (0.14.5)

Avoid specifying long messages outside the exception class

(TRY003)